.sdkmanrc

```diff
@@ -0,0 +1,3 @@
+# Enable auto-env through the sdkman_auto_env config
```

Comment:
could you please remove this file from check-in / add to your local gitignore?

Reply:
removed